<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UstawieniaARX</class>
 <widget class="QDialog" name="UstawieniaARX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>USTAWIENIA ARX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wielomian A:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wielomian B:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Opóźnienie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Szum:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leA">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leB">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinSzum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Limity</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkLimity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Limity aktywne</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Umin:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Umax</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ymin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ymax</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinUmin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinUmax">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinYmin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinYmax">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnuluj">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>540</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnZatwierdz">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Zatwierdź</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
